azure-functions/acbs-function/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ const baseParserOptions = {
 };
 
 const baseRules = {
+  'no-param-reassign': ['error', { props: true, ignorePropertyModificationsForRegex: ['req', 'res'] }],
```

Comment:
Do we need this? I can't see any new param reassigns with req/res

Reply:
good catch, removed